package.json

```diff
@@ -127,6 +127,7 @@
     "transform": {
       "^.+\\.js$": "<rootDir>/packages/babel-jest"
     },
+    "setupTestFrameworkScriptFile": "<rootDir>/test_setup_file.js",
```

Comment:
@SimenB can you explain why we need this here if we moved to `setupTestFramework` below?

Reply:
I'm not really sure what the issue is, tbh... @thymikee